resolwe_bio/processes/import_data/differential_exp.yml

```diff
@@ -84,6 +84,18 @@
           value: mirna20
         - label: mirna30 (Affy)
           value: mirna30
+    - name: case
```

Comment:
Maybe check in the beginning of the process algorithm that if cases are given, controls should be given too.